index.html

```diff
@@ -2769,47 +2803,58 @@ <h3 id="applying">
     </section>
     <section>
       <h2>
-        Image object and its members
+        ImageResource and its members
       </h2>
       <div class="issue" data-number="361"></div>
+      <pre class="idl">
+        [NoInterfaceObject]
+        dictionary ImageResource {
+          USVString src;
+          DOMString sizes;
```

Comment:
required?

Reply:
I don't think this is required.

Reply:
src is required at least. Would you assume it is "any" if sizes is not set? Maybe file an issue?